README.md

```
@@ -576,35 +576,41 @@ npx @mulmocast/slide extend init
 yarn cli extend init
 ```
 
-This copies the skill files to `.claude/skills/extend/` in your project directory.
+This copies the skill files to `.claude/skills/` in your project directory.
 
 ### Usage
 
-In Claude Code, use the `/extend` command:
+In Claude Code, use the `/narrate` command with any supported source file:
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)